tekton/pipeline_ref.go

```diff
@@ -0,0 +1,43 @@
+package tekton
+
+import tektonv1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
+
+type PipelineRef struct {
+	Name        string      `json:"name,omitempty"`
+	ResolverRef ResolverRef `json:"resolverRef"`
+}
+
+type ResolverRef struct {
+	Resolver string  `json:"resolver"`
+	Params   []Param `json:"params"`
+}
+
+type Param struct {
+	Name  string `json:"name"`
+	Value string `json:"value"`
+}
+
+func (pr PipelineRef) ToTektonPipelineRef() tektonv1.PipelineRef {
+	tektonParams := tektonv1.Params{}
```

Comment:
You can call it just `params`. Not used in this context and doesn't seem confusing to me.

Reply:
Done.